packages/ts/hilla-file-router/src/vite-plugin/collectRoutesFromFS.ts

```diff
@@ -97,6 +97,10 @@ export default async function collectRoutesFromFS(
           children: [],
         });
       }
+    } else {
+      logger.warn(
```

Comment:
Should not log for files prefixed with `_` regardless of the extension. I'm also not sure if we should log the warning also for e.g. .css files inside `views/`

Reply:
Changed to show warning only for js and ts.

Reply:
Based on how I read the code, it would still log a warning for `_foo.js` and that's not right since `_` files are supposed to be ignored regardless of the extension

Reply:
Ah, I forgot to commit this. Thanks, added.